### ISSUE TITLE
Create documentation for MyGrad

### ISSUE THREAD

**Issue by meowklaski** [rsokl/MyGrad#11]
I have begun working on using sphinx to create a docs page for mygrad. Progress is being made on the [docs](https://github.com/rsokl/MyGrad/tree/docs) 

## To-do
- [x] Create a docs README.md so people know how to build the docs
- [x] Add example section to every function's docstring (in-progress)
- [x]  A basic introduction to MyGrad, including its purpose to be a simple, but optimized autograd library. 
- [x] Demonstrate the basic usage of `mygrad.Tensor`, and its numpy-array-esque behavior (including basic and advanced indexing)
- [ ] Summarize the library's neural network tools  and discuss MyNN
- [x] Demonstrate ability to visualize computational graph
- [ ] Provide simple gradient descent example
- [ ] Document how MyGrad's back-propagation system works
- [x] Add Latex equations for neural network functions (multiclass hinge still needs docs)

**Comment by rsokl** [comment]
@davidmascharka Any interest in helping with the latex equations? (Last bullet point)

**Comment by davidmascharka** [comment]
Yeah I can contribute there. What are you envisioning? Like:

![img](https://user-images.githubusercontent.com/5611862/48630582-9fc44980-e98a-11e8-83e8-8f5784ab8a2d.png)

or something else?

**Comment by rsokl** [comment]
Opened #99 so it is easier for us to review code etc

**Comment by rsokl** [comment]
Closing this because it is so stale. Docs are viable as-is; I can circle back to make a docs-improvement wish list
